doc/man/hirte-agent.conf.5.md

```diff
@@ -92,6 +92,11 @@ LogLevel=DEBUG
 LogTarget=journald
 LogIsQuiet=false
 ```
+## FILES
+
+Distributions provide the __/usr/share/hirte-agent/config/hirte-default.conf__
+file which defines hirte-agent configuration defaults. Administrators can copy
+this file to __/etc/hirte/agent.conf__ to specify their own configuration.
```

Comment:
Well, it's not needed to copy now, as `/etc/hirte/agent.conf` contains the same set of options as `/usr/share/hirte-agent/config/hirte-default.conf`, just those options with reasonable default value are commentted out to be able to see what is the default value.
But yeah, if we add some new configuration option and the user modified `/etc/hirte/agent.conf`, this new option will be added only to `/usr/share/hirte-agent/config/hirte-default.conf` and rpm will create  `/etc/hirte/agent.conf.rpmnew` with those new additions. If the user didn't change  `/etc/hirte/agent.conf`, then it will be overwritten during package upgrade.

Reply:
Hmm, could you please check that you are using the right repo to consume hirte RPMs from?

https://github.com/containers/hirte#testing

I've just check the latest build, which is 4 days old and hirte-agent package contains the correct configuration files ...

Reply:
I am just using packages that are in Fedora 38

Reply:
Can we cut a new release for Fedora then.  Lets cut hirte-0.1.2